package.json

```diff
@@ -54,6 +54,10 @@
     "json-stable-stringify": "^1.0.0",
     "levn": "^0.3.0",
     "lodash": "^4.0.0",
+    "lodash.clonedeep": "^3.0.1",
```

Comment:
We already include lodash as a whole, so there's no need to include individual methods.